{{ cookiecutter.project_shortname }}/.travis.yml

```diff
@@ -24,12 +24,12 @@ python:
 before_install:
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
-  - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
-  - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=min setup.py > built-requirements-lowest.txt"
```

Comment:
In a defense of "built".. alternative name I considered was "compiled-requirements-*", so the already completed action (compiled/build/collected). "build" would imply that the requirements themselves are used to build something, which I'm not sure is entirely true in this case (they are more runtime dependencies, rather than compile/build ones).